sycl/include/CL/sycl/ONEAPI/reduction.hpp

```diff
@@ -53,6 +53,22 @@ using IsReduOptForFastAtomicFetch =
                    sycl::detail::IsBitAND<T, BinaryOperation>::value)>;
 #endif
 
+// This type trait is used to detect if the group algorithm reduce() used with
```

Comment:
It looks like the comment was copy-pasted from wrong place. It mentions grou algorithm reduce, while should say about atomic operations.

Reply:
Apologies and thanks for pointing this out.  I have made a change.